docs/unified-test-documentation/dasharo-compatibility/362-coreboot-customizer.md

```diff
@@ -0,0 +1,93 @@
+# Dasharo Compatibility: DMI editing tool
+
+## DMI001.001 Change the UUID
+
+**Test description**
+
+The coreboot_customizer tool is a part of the Dasharo Tools Suite. It allows to
+customize DMI information in a Dasharo .rom image. In this test case we will
+try to change the UUID.
```

Comment:
```suggestion
This test case verifies that the UUID encoded in the DMI table of an
image can be changed using our coreboot_customizer utility.
```

let's keep consistency with other test descriptions.

Reply:
Fixed in [recent commit](https://github.com/Dasharo/docs/commit/2f2c60da5365e1c65115c59cf2a341885221995f)